bermuda/factory.py

```diff
@@ -125,3 +138,38 @@
     wraps(wide_data_frame_to_triangle)(wide_data_frame_to_triangle)
 )
 Triangle.from_json = staticmethod(wraps(json_to_triangle)(json_to_triangle))
+
+# plots
+Triangle.plot_right_edge = wraps(plot_right_edge)(
```

Comment:
I think we only did this with the utils to maintain backwards-compatibility. Since these are new I think it's cleaner to just add them directly as methods. The `wraps` trick doesn't map the function signature and docstring correctly (self get's messed up), and static type checkers don't handle it correctly. Also it's just surprising to have the function implemented elsewhere.